include/types/resources.class.php

```diff
@@ -30,9 +30,9 @@ class resources extends AbstractType {
     public $resource;
 
     /**
-     * @var dateTime
+     * @var array[1, unbounded] of (object)package
```

Comment:
content model is ```package*```, meaning there can be no packages or unlimited packages, you have specified that at least one package must exist